docs/pages/application-access/getting-started.mdx

```diff
@@ -117,43 +161,23 @@ $ sudo teleport configure \
   --app-uri=http://localhost:3000
 ```
   
-Make sure to update `--app-name` and `--app-uri` accordingly if you're using your
-own web application.
-
-The `--token` flag points to the file on the Application Service host where we
-stored the token that we generated earlier.
+- Set the `--proxy` command-line option to your Teleport Proxy Service address 
+(for example, `teleport.example.com`) or Teleport Cloud tenant 
+(for example, `mytenant.teleport.sh`).
+- Set the `--token` command-line option to the file location on the Application 
+Service host where you saved the invitation token that you generated earlier.
```

Comment:
Would there be an Application Service host if you're running the service via the `teleport-kube-agent` chart? Would it make sense to make this value the value of the token instead?